packages/apps/docs/src/utils/createSlug.ts

```diff
@@ -1,11 +1,20 @@
-export const createSlug = (
-  str?: string,
-  index?: number,
-  parentTitle = 'menu',
-): string => {
+const createSlugHash = (str: string): string => {
+  let hash = 0;
+  // if the length of the string is 0, return 0
+  if (str.length === 0) return `${hash}`;
+
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  str.split('').forEach((_char, idx) => {
+    const ch = str.charCodeAt(idx);
+    hash += ch;
+  });
+  return `h${hash}`;
+};
```

Comment:
this change will introduce back the same issue we had before in the aside menus. https://alpha-docs-git-feat-docsimprovesearch2-kadena-js.vercel.app/docs/pact/reference/functions/operators#h94

Operators `!=` and `^` both shares the same hash now.
`and` `and?` shares the same hash etc..